src/modules/submissions.js

```diff
@@ -1705,9 +1705,11 @@
 				function prefillBiographyDetails() {
 					var titleParts;
 
-					// Prefill `LastName, FirstName` for Biography if the page title is two words and
+					// Prefill `LastName, FirstName` for Biography if the page title is two words
+					// after removing any trailing parantheticals (likely disambiguation), and
 					// therefore probably safe to asssume in a `FirstName LastName` format.
-					titleParts = afchSubmission.shortTitle.split( ' ' );
+					title = afchSubmission.shortTitle.replace(/ \([\s\S]*?\)$/g, '')
```

Comment:
Please run `eslint --fix .`. Style convention in this project is to haves spaces before opening and closing parentheses.